src/picterra/base_client.py

```diff
@@ -38,8 +40,19 @@ class _RequestsSession(requests.Session):
     def __init__(self, *args, **kwargs):
         self.timeout = kwargs.pop("timeout")
         super().__init__(*args, **kwargs)
+        version = pkg_resources.require("picterra")[0].version
```

Comment:
Searching a bit on the web, it looks like pkg_resources is gonna be removed in 3.12 ?

I think importlib has an easier way to figure out the version:
```
(example) $ python
from importlib.metadata import version  
version('wheel')  
'0.32.3'
```

(I think we should catch `PackageNotFoundError` and fill with no_version if that call fails for whatever reason. This seem like one thing that should be failsafe)

Reply:
Done, also added a bunch of commits before the main one